ansible/playbooks/monitoring.yml

```diff
@@ -202,13 +202,13 @@
                 description: "There are more than 1000 deprioritized crates in the docs.rs queue, and the situation didn't resolve itself in the 24 hours. The build queue is available at https://docs.rs/releases/queue"
 
             - alert: HighCpuUsage
-              expr: avg(rate(node_cpu_seconds_total{job="node",instance="docsrs.infra.rust-lang.org:9100",mode="idle"}[1m])) < 0.1
+              expr: avg(rate(node_cpu_seconds_total{job="node",instance="docsrs.infra.rust-lang.org:9100",mode="idle"}[10m])) < 0.15
               for: 30m
               labels:
                 dispatch: docsrs
               annotations:
                 summary: "High CPU usage on the docs.rs server"
-                description: "user-mode CPU usage is > 90% for longer than 30 minutes, something is wrong. Please check `htop` for which threads consume the CPU. Restarting the server helps then."
+                description: "user-mode CPU usage is > 85% for longer than 30 minutes, something is wrong. Please check `htop` for which threads consume the CPU. Restarting the server helps then."
```

Comment:
Am I understanding the syntax correctly in that this is measuring that the 10-minute rolling average idle time over a 30-minute period is always under 15%? 

If so, I think the description could use a bit of adjustment. The description makes it sound like CPU usage *never* dips below 85% in the 30-minute window, but I believe it can temporarily.

Reply:
I updated the descrition